spec/serializer/defence_organisation_serializer_spec.rb

```diff
@@ -15,15 +15,13 @@
                     postcode: "EC4A 2AH")
   end
 
-  context "with attributes" do
-    let(:attribute_hash) { subject[:data][:attributes] }
+  let(:attributes) { subject[:data][:attributes] }
 
```

Comment:
Sorry to be a pedant, but just for completion, could we have 
`it { expect(attributes[:id]).to eq("UUID") }`

Or even 11223344 if you want to change the double too